lib/nlp/nlp-excel-reader.js

```diff
@@ -81,6 +81,7 @@ class NlpExcelReader {
         row.language,
         row.intent,
         row.response,
+        row.url,
```

Comment:
row.url should go after row.condition given the interface of the function. That's what cause the error in tests.

Reply:
ok, fixed it. thanks!